modules/ROOT/pages/runtime-fabric-setup.adoc

```diff
@@ -22,12 +22,12 @@ Anypoint Visualizer relies on a header injection to display connections between
 Anypoint Visualizer requires an additional header injection for Anypoint Runtime Fabric deployments. However, due to a potential performance impact in Mule 4, header injection is disabled by default.
```

Comment:
Is "header injection" typically a countable noun ("a water drop") or a noncountable noun ("water")?
I'm just wondering if we should be using "an additional header injection"....
Ditto for "a potential performance impact".

Reply:
```suggestion
Anypoint Visualizer requires an additional header injection for Anypoint Runtime Fabric deployments. However, due to potential performance impact in Mule 4, header injection is disabled by default.
```